lms/static/scripts/frontend_apps/api-types.ts

```diff
@@ -302,9 +307,36 @@ export type StudentsResponse = {
   pagination: Pagination;
 };
 
+export type StudentGradingSyncStatus = 'in_progress' | 'finished' | 'failed';
+
+export type StudentGradingSync = {
+  h_userid: string;
+  status: StudentGradingSyncStatus;
+};
+
+export type GradingSyncStatus = 'scheduled' | StudentGradingSyncStatus;
+
 /**
  * Response for `/api/dashboard/assignments/{assignment_id}/grading/sync`
  */
 export type GradingSync = {
-  status: 'scheduled' | 'in_progress' | 'finished' | 'failed';
+  /**
+   * Global sync status.
+   * If at least one student grade is syncing, this will be `in_progress`.
```

Comment:
What is the value of this field if a grade sync has never been started for the current assignment?

Reply:
When an assignment has never been synced, the endpoint returns a 404 response which is handled separately, so this field will always have one of these values.

Reply:
OK. It might be helpful to add a comment to that effect in the code.